src/ucs/datastruct/notifier.h

```diff
@@ -49,6 +50,7 @@ static inline void ucs_notifier_chain_call(ucs_notifier_chain_t *chain)
     ucs_notifier_chain_elem_t *elem;
 
     ucs_notifier_chain_for_each(elem, chain) {
+        ucs_memory_cpu_load_fence();
```

Comment:
I want to allow adding (but not removing) elements to the call chain from progress thread, while going over the array from main thread.